### PR TITLE
Rename to concord and prepare to publish on npm

### PR DESCRIPTION
**Pull request by bergundy**

### CODE REVIEW

**Comment by bergundy** [comment]
@michaeladda I set the license to MIT.
WDYT?

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "launch",
+  "name": "concord",
   "version": "0.0.1",
```
Comment (vogre) [inline package.json:3]: Do we want a more `semver`-useful version?

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (arielshaqed) [inline LICENSE:1]: I reckon having this file means @michaeladda should approve.

**Review by arielshaqed — COMMENTED** [review]
`.gitignore` should ignore `dist`.

File: package.json [hunk 3]
```diff
@@ -8,24 +8,25 @@
...
+    "json-schema",
+    "typescript",
     "schema",
-    "methods"
+    "methods",
+    "rpc"
   ],
   "author": "bergundy",
-  "license": "ISC",
+  "license": "MIT",
```
Comment (vogre) [inline package.json:25]: We prefer MIT to the default?

**Review by vogre — APPROVED** [review]
Thanks!

Good to be rid of `dist/` in repo.

**Review by michaeladda — APPROVED** [review]
Looks great!
MIT is nice.
Do you want to bump the version or keep it at 0.0.1?

**Comment by bergundy** [comment]
We're staying with the version.
I'm changing the cli to support different runtimes before publishing

**Pull request merged**